core/geometry/src/curve/Arc3d.ts

```diff
@@ -445,6 +446,36 @@ export class Arc3d extends CurvePrimitive implements BeJSONFunctions {
     }
     return LineSegment3d.create(start, end);
   }
+  /**
+   * Create a circular arc defined by start and end points and radius.
+   * @param start start point of the arc
+   * @param end end point of the arc
+   * @param helper a third point near the arc in its plane, or a vector in the direction of the arc normal
+   * @returns the constructed arc, or line segment if desired arc cannot be constructed
+   */
+  public static createCircularStartEndRadius(start: Point3d, end: Point3d, radius: number, helper: Point3d | Vector3d): Arc3d | LineSegment3d {
+    const semiChordLen2 = 0.25 * start.distanceSquared(end);
```

Comment:
When I implemented `createCircularStartTangentEnd` I added couple lines of doc (inside the function body) explaining the algorithm I used. It makes it much easier to read and understand the function body. Please do the same here.

Reply:
Done!